### PR TITLE
Adding upload-control

### PR DESCRIPTION
**Pull request by Rynx724**
Adding upload-control :D

### CODE REVIEW

**Comment by olmedo90** [comment]
Captura de pantalla (777)

Es ACORDE al figma propuesto. like

File: src/ui/components/uploadControl/uploadControl.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React, { useCallback, useState } from "react";
```
Comment (Ditmar) [inline src/ui/components/uploadControl/uploadControl.jsx:1]: use single quotes

File: src/ui/components/uploadControl/uploadControl.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import { useDropzone } from "react-dropzone";
```
Comment (Ditmar) [inline src/ui/components/uploadControl/uploadControl.jsx:2]: use single quotes

File: src/ui/components/uploadControl/uploadControl.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+import "./index.css";
```
Comment (Ditmar) [inline src/ui/components/uploadControl/uploadControl.jsx:3]: use single quotes

File: src/ui/components/uploadControl/uploadControl.stories.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
```
Comment (Ditmar) [inline src/ui/components/uploadControl/uploadControl.stories.js:1]: use single quotes

File: src/ui/components/uploadControl/uploadControl.stories.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+import { UploadControl } from "./uploadControl";
```
Comment (Ditmar) [inline src/ui/components/uploadControl/uploadControl.stories.js:2]: use single quotes

**Review by Ditmar — COMMENTED** [review]
Great job, please only one thing use a simple quotes

**Review by Ditmar — APPROVED** [review]
good job

**Pull request merged**